### PR TITLE
Update pack tags

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25379
related: https://github.com/demisto/etc/issues/24069

## Description
tags cleanup for packs.

### CODE REVIEW

File: Packs/Compliance/pack_metadata.json [hunk 1]
```diff
@@ -8,13 +8,8 @@
...
     "categories": [],
     "tags": [
-        "Compliance",
         "Breach",
-        "GDPR",
-        "US",
-        "Notification",
-        "Breach notification",
-        "Data breach"
+        "GDPR"
```
Comment (altmannyarden) [inline Packs/Compliance/pack_metadata.json:12]: Please add HIPAA too

File: Packs/BreachNotification-US/pack_metadata.json [hunk 2]
```diff
@@ -7,11 +7,7 @@
     "url": "https://www.paloaltonetworks.com/cortex",
     "email": "",
     "categories": [],
-    "tags": [
-        "Breach Notification",
-        "PII",
-        "Compliance"
-    ],
+    "tags": [],
```
Comment (altmannyarden) [inline Packs/BreachNotification-US/pack_metadata.json:10]: Tags to add: Breach, Compliance, PII

**Comment by yaakovi** [comment]
@altmannyarden @michalgold 
updated the tags according to review. please go over it again.

**Comment by yaakovi** [comment]
failures unrelated to tags updates

**Pull request merged**